submissions/maryOkosun/dommanipulation/product.js

```diff
@@ -0,0 +1,59 @@
+let quantity = document.getElementById("quantity"),
+    price = document.getElementById("price"),
+    shippingCost = document.getElementById("shippingCost"),
+    totalCost = document.getElementById("totalCost"),
+    button = document.getElementById("buy-btn"),
+    quantityRegex = /^[1-9]+[0-9]*$/;
+
+function quantityValue(){
+    if(!(quantity.value.match(quantityRegex))){
+        quantity.nextElementSibling.innerHTML = "Please enter a valid quantity";
+        button.disabled = true;
+        shippingCost.innerHTML = 0;
+        totalCost.innerHTML = 0;
+        return false;
+    }
```

Comment:
This check `(quantity.value   < 1)` should be on the `else if` statement. There is no need written a function( i.e `checkInteger()` ) for this